stubs/tensorflow/tensorflow/train/experimental.pyi

```diff
@@ -0,0 +1,23 @@
+from _typeshed import Incomplete
+
+# @dataclasses.dataclass(frozen=True)
+# class ShardableTensor:
```

Comment:
For this one I am mainly waiting on either commented out stubs to be finished and not comment or from them to be removed and left for future pr.

Reply:
This class was [added to the tensorflow repo last month](https://github.com/tensorflow/tensorflow/blame/6dddfa1b666fab4d070052311a312f62f2739605/tensorflow/python/checkpoint/sharding/sharding_util.py#L39), so it's not in the doc yet (nor in the latest runtime version). I realized this afterwards and left them comment out to be able to easily add them in the future (I'm assuming the classes will make their way into the next version of tensorflow).
Should I add a comment explaining this, or is it better to remove them for now and add them in a future PR once they have been added to a released version of tensorflow ?

Reply:
I recommend removing them. After next tensorflow release they can be added then.

Reply:
Done in [080b155](https://github.com/python/typeshed/pull/11327/commits/080b1558d13f7f6a8b30a2522a3b5b31088fbe0f)